snowflake-table-grant-all/module_test.go

```diff
@@ -0,0 +1,20 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/chanzuckerberg/go-misc/tftest"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+)
+
+func TestModule(t *testing.T) {
+	test := tftest.Test{
```

Comment:
If you set the required input variables and set `TestMode: tftest.Init` then we can at least test that the code is valid and initializes without setting up snowflake credentials.

Reply:
https://github.com/chanzuckerberg/go-misc/blob/main/tftest/test.go#L18